src/execution/experimental-executions.js

```diff
@@ -487,28 +642,29 @@ rules.addRule(
     createPureMetric
 );
 
-function getMetricFactory(measure) {
-    const factory = rules.match(measure);
+function getMetricFactory(measure, mdObj) {
+    const factory = rules.match(measure, mdObj);
 
     invariant(factory, `Unknown factory for: ${measure}`);
 
     return factory;
 }
 
-const isDateFilterExecutable = dateFilter =>
-    get(dateFilter, 'from') !== undefined &&
-    get(dateFilter, 'to') !== undefined;
-
-const isAttributeFilterExecutable = listAttributeFilter =>
-    notEmpty(get(listAttributeFilter, ['default', 'attributeElements']));
+function isDateFilterExecutable(dateFilter) {
```

Comment:
[Q] Why is this rewritten to `function` function while others were kept in lambda way?

Reply:
all lambdas on first level rewritten to named function